src/test/java/com/automatedtest/sample/definitions/SearchImageByDropLink.java

```diff
@@ -0,0 +1,27 @@
+package com.automatedtest.sample.definitions;
+
+import com.automatedtest.sample.pages.ImageTabOfSearchResultPage;
+import com.automatedtest.sample.pages.SearchResultPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+public class SearchImageByDropLink {
```

Comment:
this is bad design as well, it's too specific for a definition class, if we doing this, we'll end up with so many definitions class at the end of the day. just name it `SearchImage` and we can put all related steps in it